{"name": "test_send_message_success[test_info0]", "status": "broken", "statusDetails": {"message": "TypeError: 'module' object is not callable", "trace": "@pytest.fixture(scope=\"class\")\n    # 设置浏览器前置条件\n    def browser():\n        \"\"\"启动浏览器和关闭浏览器\"\"\"\n        chrome_options = ChromeOptions()\n        chrome_options.binary_location = r\"C:\\Users\\Administrator\\AppData\\Local\\Google\\Chrome\\Application\\chrome.exe\"\n>       driver = webdriver.chrome(executable_path=r\"C:\\Users\\Administrator\\AppData\\Local\\Google\\Chrome\\Application\\chromedriver.exe\", options=chrome_options)\nE       TypeError: 'module' object is not callable\n\ntests\\conftest.py:17: TypeError"}, "parameters": [{"name": "test_info", "value": "{'message': '123456789测试私信', 'file': 'd:S.jpg'}"}], "start": 1594900804568, "stop": 1594900804568, "uuid": "73637a92-25f5-448e-88fd-2ff5072ac4e3", "historyId": "747e9887dd2aa681c8e525c7f5b89af3", "testCaseId": "e59a1fb7fb85fdb0cc76174ca11774d1", "fullName": "tests.test_06private_message.TestPrivateMessage#test_send_message_success", "labels": [{"name": "tag", "value": "success"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_06private_message"}, {"name": "subSuite", "value": "TestPrivateMessage"}, {"name": "host", "value": "USER-20180313JZ"}, {"name": "thread", "value": "6928-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_06private_message"}]}